mongo/integration/mtest/csfle_not_enabled.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) MongoDB, Inc. 2022-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+//go:build !cse
+// +build !cse
+
+package mtest
+
+// CSFLEEnabled returns true if driver is built with Client Side Field Level Encryption support.
+// Client Side Field Level Encryption support is enabled with the cse build tag.
+func CSFLEEnabled() bool {
```

Comment:
Optional: Functions that don't modify anything and return only a `bool` conventionally have "is" or "has" in the name. Consider updating to the slightly more conventional `IsCSFLEEnabled()`.

Reply:
Done.